rdr_service/resource/schemas/participant.py

```diff
@@ -273,6 +282,20 @@ class ParticipantSchema(Schema):
     # PDR-178:  Add CABoR authored to participant top-level schema
     cabor_authored = fields.DateTime()
 
+    deceased_status = fields.EnumString(enum=DeceasedStatus)
```

Comment:
Not an issue but perhaps helpful for housekeeping / maintenance:  it could help if the field order of the newly ported fields here aligned with the rdr_service.model.BQParticipantSummarySchema class definition, just for visually verifying consistency between the two.   Provided we can reorder this field list at will.

Reply:
I would rather not do that, I would prefer to group the field definitions where ever possible.